lambda_code/src/fhir_controller.py

```diff
@@ -3,16 +3,23 @@
 import uuid
 from typing import Optional
 
+import boto3
+from botocore.config import Config
 from fhir.resources.operationoutcome import OperationOutcome
 
 from fhir_repository import ImmunisationRepository, create_table
 from fhir_service import FhirService
-from models.errors import Severity, Code, create_operation_outcome, ResourceNotFoundError, UnhandledResponseError
+from models.errors import Severity, Code, create_operation_outcome, ResourceNotFoundError, UnhandledResponseError, \
+    ValidationError
+from pds_service import PdsService, Authenticator
 
 
 def make_controller():
     imms_repo = ImmunisationRepository(create_table())
-    service = FhirService(imms_repo=imms_repo)
+    env = "internal-dev"
```

Comment:
Environment variables for this and the AWS region?

Reply:
AMB-1838 will address this. I added TODOs somewhere else in the code as a reminder.